docs/cloud/queries.md

```diff
@@ -0,0 +1,140 @@
+sidebar_label: Queries
+---
+
+# Running Queries
+
+Once you've added a connection you will be able to run [SQL queries](sql/steampipe-sql) to explore your data, either interactively in the console, or via any PostgreSQL-compatible client - see our [integrations pages](cloud/integrations/overview) for some examples.
+
+## Exploring Schemas
+
+If you navigate to your workspace, then the **Query** tab, you'll see the interactive query console.
+
+<img src="/images/docs/cloud/cloud-query-editor.png" width="400pt"/>
+<br />
+
+From here you can either explore your schemas on the left, or dive right in and test out your own queries in the editor. The schema list supports flexible searching across all the tables in your schemas. For example, if you search for `hack new`, 
+that will find a match for the `hackernews_show_hn` table in our `hackernews` schema.
+
+<img src="/images/docs/cloud/cloud-query-schema-search.png" width="200pt"/>
+<br />
+
+Clicking the search result will automatically generate a query to select all the columns from that schema table, limited to 100 rows and will automatically run it for you.
+
+<img src="/images/docs/cloud/cloud-query-table-results.png" width="400pt"/>
+<br />
+
+If you click the `Edit` button you can amend the query, perhaps by selecting just the columns you're interested in, or adding a `where` clause to filter the results. Please note that we limit queries to 5,000 rows in the interactive query console.  
+
+<img src="/images/docs/cloud/cloud-query-custom-query.png" width="400pt"/>
+<br />
+
+## Downloading Results
+
+After you've run a query, you can download the results to a CSV file by clicking the **Download** button at the bottom of the query editor.
+
+## Saving & Sharing Snapshots
+
+Steampipe Cloud allows you to save and share query **snapshots** that are dashboards containing a table generated from your query results.
+
+To take a snapshot, click the **Snap** button at the top of the query editor after you have run the query you wish to snap.  
+
+<img src="/images/docs/cloud/cloud-query-toolbar.png" width="200pt"/>
+<br />
+
+This will then take you to the dashboard snapshot view.
+
+<img src="/images/docs/cloud/cloud-query-snapshot.png" width="400pt"/>
+<br />
+
+## Browsing Snapshots
+
+From any query snapshot, you can browse and navigate its snapshot history. We generate a hash of the query used in the snapshot, allowing us to find other snapshots relating to this query. 
+Click the **View** button at the top of the dashboard to display the menu.  You can select a date from the calendar and the snapshots from that date are listed beneath. Click any snapshot to open it.
+
+<img src="/images/docs/cloud/cloud-query-related-snapshots.png" width="200pt"/>
+<br />
+
+Clicking **All matching snapshots** at the bottom of the menu will take you to the snapshots page (you can also navigate to this page by clicking the **Snapshots** button at the top of the **Dashboards** pane).
+
+
+## Managing Snapshots
```

Comment:
Do you think we should include this entire section here?  or just link to https://steampipe-cp8jwleq3-turbot.vercel.app/docs/cloud/dashboards#managing-snapshots with a closing sentence from Browsing Snaphots?   It feels a little repeated here, but I dont feel strongly about it either...

Reply:
I had the same thought when writing it. I wasn’t strong either way, but perhaps linking across would be better?

Reply:
I don't mind either way.  I'm going to approve, but if you want to try it the other way thats fine.